clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/AmiIdResolver.groovy

```diff
@@ -39,15 +42,15 @@ class AmiIdResolver {
     if (launcher) {
       req.withExecutableUsers(launcher)
     }
-    Image resolvedImage = amazonEC2.describeImages(req)?.images?.getAt(0)
+    Image resolvedImage = retrySupport.retry({ -> amazonEC2.describeImages(req)}, 5, 1000, true)?.images?.getAt(0)
```

Comment:
TBH I probably wouldn't do this exponentially ... there's already exponential backoff inside of the aws sdk.

Reply:
That makes me wonder if we should do this at all, then. If anything, seems like we should extend our backoff window configuration? 

(It's been years since I've looked at the AWS SDK backoff strategy config stuff, so IDK)

Reply:
TBH the retries may be better served inside of our sdk proxy.

I'm not sure we want to blow out the exponential backoff anymore than it already is (2^8 * 100ms on the worst interval -- so it seems like we're already retrying for a minute or so).

By doing this sort of thing inside our sdk proxy, we could let the sdk exponentially backoff a bit ... but then retry ourselves at a fixed interval.

Reply:
Sounds good. I am going to close this - probably being too reactive to a bigger issue we're encountering.